packages/react-core/src/demos/PrimaryDetail.md

```diff
@@ -567,21 +575,18 @@ class PrimaryDetailCardView extends React.Component {
                         </DropdownItem>
                       </DropdownList>
                     </Dropdown>
-                    <Checkbox
-                      checked={isChecked}
-                      onClick={(event) => this.onCheckboxClick(event, product.id)}
-                      value={product.id}
-                      isChecked={selectedItems.includes(product.id)}
-                      aria-label="card checkbox example"
-                      id={`check-${product.id}`}
-                    />
                   </>
                 )
               }}
+              selectableActions={{ isChecked: selectedItems.includes(product.id), selectableActionId: `selectable-actions-item-${product.id}`, selectableActionAriaLabelledby: `${'card-view-' + key}`, name: `check-${product.id}`, onChange: () => this.onClick(product.id) }}
             >
               <img src={icons[product.icon]} alt={`${product.name} icon`} style={{ height: '50px' }} />
             </CardHeader>
-            <CardTitle>{product.name}</CardTitle>
+            <CardTitle>
+              <Button variant="link" isInline onClick={this.onCardClick}>
+                {product.name}
+              </Button>
```

Comment:
The onCardClick may need to be tweaked a bit to get the `id` of the currentTarget's nearest Card (or the `id` from Card moved here if it's not being used for much else). The drawer content isn't getting rendered properly currently.

If we do this, we could also add an `aria-expanded` attribute on this button based on whether the `activeCard` state matches the `id`.